### PR TITLE
Fix build-and-ship.sh

### PR DESCRIPTION
**Pull request by RileyApeldoorn**
We ran into some issues with the `build-and-ship.sh` script, here are the fixes.

### CODE REVIEW

File: package/build-and-ship.sh [hunk 1]
```diff
@@ -25,6 +25,6 @@ gcloud compute scp --tunnel-through-iap "$PKGFILE" "$FREIGHT_HOST:/tmp/$PKGFILE"
 # shellcheck disable=SC2087
 gcloud compute ssh --tunnel-through-iap "$FREIGHT_HOST" -- -T <<END
 sudo freight add "/tmp/${PKGFILE}" apt/bionic apt/focal
```
Comment (bertptrs) [inline package/build-and-ship.sh:27]: This command should also run as the `freight` user. It doesn't cause problems directly but can interfere with cleaning up packages later.

**Review by bertptrs — APPROVED** [review]

**Comment by RileyApeldoorn** [comment]
@OpsBotPrime merge

**Comment by OpsBotPrime** [comment]
Pull request approved for merge by @RileyApeldoorn, rebasing now.

**Comment by OpsBotPrime** [comment]
Rebased as f2cd083d38caf9fa643c82d9173dd488ed4ce94a, waiting for CI …

**Pull request merged**